dashboard/views.py

```diff
@@ -312,7 +312,7 @@ def assignment_view(request, course_id=0):
     week_list = []
     for item in assignment_list:
         week_list.append(item['week'])
-    weeks = set(week_list)
+    weeks = sorted(set(week_list))
```

Comment:
Why is there a new set variable here? Seems like it should have just been
week_list.sort()

And use week_list later in the for loop.

```suggestion
    week_list.sort()
```

And changing
`for week in week_list:`

Reply:
Is this just a set to remove duplicates?

Reply:
Yes to remove the duplicated it is a `Set`

Reply:
Yeah, so then using set from the beginning probably would have made the code a little easier to read. https://github.com/tl-its-umich-edu/student-dashboard-django/compare/master...jonespm:issue_261-rewrite?expand=1

Yeah, minor cleanup but this file could probably a little more cleanups.

Reply:
Also another cool python function is [enumerate](http://book.pythontips.com/en/latest/enumerate.html) which I changed the for loop below to use
https://github.com/tl-its-umich-edu/student-dashboard-django/commit/54afbb42ce6edb451ecf5a3aa5c38bf9bfe922c5